### ISSUE TITLE
FEATURE request - Lock some item in the history to keep them from rolling out

### ISSUE THREAD

**Issue by v1ncen72** [snark/jumpcut#57]
Hi there,

It would be very great to be able to lock items in the drop down menu to prevent them from disappearing. For instance, my email signature is something I use constantly, but in most case I still need to go back to Evernote to copy it because it disappears from the list.

Having a small icon on each item line with a lock to allow the user to keep the items on the list would be very useful.

**Comment by snark** [comment]
This is something that will require a fairly extensive set of revisions, but I can see the utility. I've been contemplating some broader revisions to Jumpcut—use of sqlite3 as the underlying data store rather than a plain-text file, most notably—so this would likely be something bundled into that. I don't anticipate this happening in the immediate future but it's probably something good for the long-term health of the application, so I'd like to do it!

**Comment by Vadorequest** [comment]
I come from Ditto and they managed it differently, they always "put back on top" what was used last.

it's a different feature, but with a similar goal, and which might be much simpler to implement (and maybe even what you're really looking for!)

**Comment by v1ncen72** [comment]
@Vadorequest 
Indeed, this would be perfect, it is exactly what I am looking for. Now if only Ditto could run on Mac OS...

**Comment by kailynnelson** [comment]
Every time I use Jumpcut, I wish this was a thing 🙏🏼

**Comment by snark** [comment]
I've added a new feature that pushes items to the top on use, which should help but doesn't directly resolve this use case -- I'm now planning the UX around how this might work more broadly.
